tests.sh

```diff
@@ -111,7 +127,8 @@ start_agent(){
 #Dump attach info
 dump_attach_info(){
     echo -e "\nCMD: Dump attach info file...\n"
-    cmd="daos_agent -i -o $DAOS_AGENT_YAML dump-attachinfo -o Log/$SLURM_JOB_ID/daos_server.attach_info_tmp"
+    #cmd="daos_agent -i -o $DAOS_AGENT_YAML dump-attachinfo -o Log/$SLURM_JOB_ID/daos_server.attach_info_tmp"
```

Comment:
Why is this replaced with the cmd below?
I think this will break selftest.
Make sure you run selftest as well whenever you make script changes to avoid regression.